.codeclimate.yml

```diff
@@ -6,3 +6,7 @@ engines:
         enabled: false
       require-jsdoc:
         enabled: false
+exclude_patterns:
+- "bin/render-pdf.js"
+- "db/migrate/20110919012833_create_pages.rb"
```

Comment:
I dont know if we want to run the checks across migrations for any reason so I've disabled just these two for now as they were throwing duplicated code errors.